app/sgqlc.py

```diff
@@ -2,47 +2,50 @@
 
 from fastapi import HTTPException
 
+from app.data_schema import *
 from sgqlc.operation import Operation
 from app.sgqlc_schema import Query
 
 
-BAD_REQUEST = 400
-UNAUTHORIZED = 401
-NOT_FOUND = 404
-METHOD_NOT_ALLOWED = 405
-INTERNAL_SERVER_ERROR = 500
-
-
 class SimpleGraphQLClient:
     def add_count_field(self, item, query):
         # Add default count field to query
         count_field = f"total: _{item.node}_count"
         if item.filter != {}:
             # Manually modify and add count filed into graphql query
             filter_argument = re.sub(
-                '\'([_a-z]+)\'', r'\1', re.sub('\{([^{].*[^}])\}', r'\1', f'{item.filter}'))
+                '\'([_a-z]+)\'', r'\1', re.sub(r'\{([^{].*[^}])\}', r'\1', f'{item.filter}'))
             count_field = re.sub(
                 '\'', '\"', f'total: _{item.node}_count({filter_argument})')
         return query + count_field
 
+    def update_manifests_information(self, query):
```

Comment:
You can find under this function. These are used to fetch different type of manifests file.